www/board/agenda/data/reminder1.txt

```diff
@@ -39,21 +39,10 @@ report should look like can be found at:
 
   https://www.apache.org/foundation/board/reporting
 
-The report should be committed to the meeting agenda in the board directory
-in the foundation repository, trying to keep a similar format to the others.
-This can be found at:
-
-  https://svn.apache.org/repos/private/foundation/board
-
-Reports can also be posted using the online agenda tool:
+Reports should be posted using the online agenda tool:
 
   [agenda][link]
 
-Your report should also be sent in plain-text format to [whoTo]
-with a Subject line that follows the below format:
-
-  Subject: [REPORT] [project] - [month] [year]
-
 Cutting and pasting directly from a Wiki is not acceptable due to formatting
 issues. Line lengths should be limited to 77 characters.
```

Comment:
Perhaps you could note here that whimsy allows you to reflow lines, so it's ok to copy/paste from another source as long as you reflow the report in whimsy before committing.

Same comment on the reminder2 below.